server/config.go

```diff
@@ -47,7 +47,7 @@ func (cfg *Config) Check() error {
 
 	// provide dummy values to eigenda client config. Since the client won't be called in this
 	// mode it doesn't matter.
-	if cfg.MemstoreEnabled {
+	if cfg.MemstoreEnabled || !cfg.VerifierConfig.VerifyCerts {
```

Comment:
This won't be nice for users, because the eigenda_client constructor right now doesnt do sanitization of its input (it should be it doesnt).

Can you instead change to crashing with an error if svcManagerAddr and EthRpcUrl are not set when memstoreEnabled=false?

Reply:
https://github.com/Layr-Labs/eigenda-proxy/blob/main/server/config.go#L48-L60
Think this should be changed instead, to pull out the checks outside of the if statement, since eth_rpc and svc_manager_addr are no longer related to verifyCerts, they are always needed WHEN memstore is disabled.

Reply:
[b3dd92b](https://github.com/Layr-Labs/eigenda-proxy/pull/205/commits/b3dd92b13ac252e7e5681234e481e5c7aa3fc3e3)